olx.py

```diff
@@ -0,0 +1,386 @@
+# python modules
+import json
+from urllib.parse import urlparse
+import logging
+
+# third party
+import requests
+from bs4 import BeautifulSoup
+
+# own modules
+from scrapper_helpers.utils import caching
+
+BASE_URL = 'https://www.olx.pl/'
+OFFERS_FEATURED_PER_PAGE = 3
+POLISH_CHARACTERS_MAPPING = {"ą": "a", "ć": "c", "ę": "e", "ł": "l", "ń": "n", "ó": "o", "ś": "s", "ż": "z", "ź": "z"}
+DEBUG = True
```

Comment:
at the very top:
`import os`
and then here:
`DEBUG = os.environ.get('DEBUG')`